### DISCUSSION TITLE
Basic gray palette as laser power

### DISCUSSION THREAD

**Discussion by Milan-As** [arkypita/LaserGRBL#1392] [Ideas]
I would welcome the possible selection function to load SVG. The point is, the curves of small objects cannot be engraved quickly (they will burn out) and the laser power should be reduced. Leave 100% power for longer straight vectors.
The idea is that the SVG would take into account the color of the object. In the case of the basic gray palette (Inkspace), the laser power would be controlled proportionally to the gray color (gray 50%> laser 50% ...).

**Comment by Stevyjb** [comment]
Hi, Yes I have had this problem with SVG files of Text when the letters with curves are over-burnt . There seems to be a problem how this software handles Curves in SVG files
